src/views/Timeline.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <amb-app-frame>
+    <ConnectionStatusSnack
```

Comment:
@vickywane I think the connectivity snackbar should be at the application level as is the nav bar. Not only tied to the timeline page. This is the main intended use of `v-snackbar` vs `v-label` which is intended at a local component level.

Reply:
Alright. I have moved the Snacbar into the `App.vue` component